lite/examples/object_detection/android/app/src/main/java/org/tensorflow/lite/examples/detection/LegacyCameraConnectionFragment.java

```diff
@@ -178,6 +147,43 @@ private void stopBackgroundThread() {
       LOGGER.e(e, "Exception!");
     }
   }
+  
+  private void startCamera(final SurfaceTexture texture) {
```

Comment:
Any specific reason to take the SurfaceTexture as a parameter?
This is an instance method, so might as well access the `availableSurfaceTexture` directly.

Reply:
Nice suggestion. I have removed this parameter in one additional commit in this pull request.